firefox-ios/Client/Frontend/ShortcutsLibrary/ShortcutsLibraryState.swift

```diff
@@ -0,0 +1,76 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Common
+import Redux
+
+struct ShortcutsLibraryState: ScreenState, Equatable {
+    var windowUUID: WindowUUID
+    let topSitesData: [TopSiteConfiguration]
```

Comment:
nit: to be consistent with the new naming?
```suggestion
    let shortcuts: [TopSiteConfiguration]
```